### PR TITLE
Homepage finished all articles displayed in different route

### PR DESCRIPTION
**Pull request by Onuryilmaz24**

### CODE REVIEW

File: src/Components/AllArticles.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  const handleNextClick = (e) => {
+    e.preventDefault();
+
+    setPage((currPage) => {
+      currPage += 1;
+      return currPage;
+    });
+  };
+  const handlePreviousClick = (e) => {
+    e.preventDefault();
```
Comment (ghost) [inline src/Components/AllArticles.jsx:22]: What is this for? What behaviour are you trying to prevent?

File: src/Components/AllArticles.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  useEffect(() => {
+    setLoading(true);
+    getAllArticles(10, page).then((data) => {
+      setAllArticles(data);
+      setLoading(false);
+    });
+  }, [page]);
+
+  console.log(page)
```
Comment (ghost) [inline src/Components/AllArticles.jsx:43]: Don't leave in console logs.

File: src/Components/HomeArticles.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  useEffect(() => {
+    getAllArticles(articlesToShow, 1)
+      .then((data) => {
+        setArticles(data);
+      })
+      .catch((error) => {
+        console.log("FetchError", error);
+      });
+
+    window.addEventListener("resize", updateArticlesToShow);
```
Comment (ghost) [inline src/Components/HomeArticles.jsx:31]: I don't believe this is working as you'd hoped as your page is not responsive

**Review by ghost — COMMENTED** [review]
This looks like a nice start. Your page however is not responsive and does not change when the window is smaller, nor does it fit on a mobile screen.

File: src/Components/HomeArticles.jsx (hunk 3 above)
Comment (Onuryilmaz24) [reply src/Components/HomeArticles.jsx:31]: Actually When I reduce my with to 1024 at home page it displays 3 cards in box less then 768px it displays 2 cards.

**Pull request merged**